src/site/content/en/fast/fast.11tydata.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
           'what-is-speed',
           'how-to-measure-speed',
           'how-to-stay-fast',
+          'measure-performance-with-the-rail-model',
```

Comment:
Let's set the URL to `rail`. This is the authoritative doc on the topic so I'm fine with using `web.dev/rail`.